src/components/GameCard.tsx

```diff
@@ -22,12 +25,34 @@ interface GameCardProps {
     notes?: string;
     platformsOwnedOn?: Array<string>;
     dateAdded?: Date;
+}
 
+interface LibraryOption {
+    label: string,
+    value: string
 }
 
 export function GameCard(gameCardProps : GameCardProps ) {
-    // will eventually be an API function
-    const addToLibrary = () => true;
+    const [libraryOptions, setAllLibraryOptions] = useState<Array<LibraryOption>>([])
+    const [selectedLibrary, setSelectedLibrary] = useState("")
+
+    useEffect(() => {
+        const fetchAllLibraries = async () => {
+            const libraries = await libraryApi.getAllLibraries()
+            const libraryOptions = new Array<LibraryOption>()
+            libraries.forEach((library) => {
+                libraryOptions.push({ label: library.name, value: library.id })
+            } )
+            setAllLibraryOptions(libraryOptions)
+        }
+        fetchAllLibraries()
+    }, [])
+
+    const addToLibrary = async (gameId: string, libraryId: string) => {
+        // todo: need to check if it's an instance or game
+        await libraryApi.addToLibrary(gameId, libraryId)
+        console.log("Successfully added to library")
```

Comment:
Do we need to keep this log?

Reply:
Probably not, it is useful for debugging though.